dev.ters.Chordcat.yml

```diff
@@ -0,0 +1,63 @@
+id: dev.ters.Chordcat
```

Comment:
how do I know you own ters.dev? Can you link the github somewhere on the webpage?

Reply:
My name is here on https://ters.dev/

"Shriram R"

I have also added a TXT record to the dns of chordcat.ters.dev linking to the github repo

you can check it here https://www.nslookup.io/domains/chordcat.ters.dev/dns-records/txt/

or via the command line

```
shriram@think182:~$ dig @ns-cloud-e4.googledomains.com. chordcat.ters.dev. TXT

; <<>> DiG 9.18.28-1~deb12u2-Debian <<>> @ns-cloud-e4.googledomains.com. chordcat.ters.dev. TXT
; (2 servers found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 44832
;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
; COOKIE: cae9b295a0b1f37701318f7ce87aec42f5862f7f68b37c0238a0659c72 (good)
;; QUESTION SECTION:
;chordcat.ters.dev.		IN	TXT

;; ANSWER SECTION:
chordcat.ters.dev.	14400	IN	TXT	"https://github.com/shriramters/chordcat"

;; Query time: 40 msec
;; SERVER: 216.239.38.110#53(ns-cloud-e4.googledomains.com.) (UDP)
;; WHEN: Thu Oct 31 17:36:18 IST 2024
;; MSG SIZE  rcvd: 131
```

Reply:
thanks